components/LibraryButton/index.js

```diff
@@ -1,9 +1,14 @@
 import Link from "next/link";
+import { StyledLibraryButton } from "./StyledLibraryButton";
 
 export default function LibraryButton() {
   return (
-      <Link href={`/profile/library`}>
-  <button>Library</button>
-  </Link>
+      
+  <StyledLibraryButton>
+    <Link href={`/profile/library`}>
+         My Library 
+         </Link>
+        </StyledLibraryButton>
```

Comment:
From a semantic point of view, you wouldn't use a `button` to link between pages. A `button` is there to do some kind of interaction, an `a`-tag is for navigation between pages.